doc/usage.md

```diff
@@ -311,6 +311,8 @@ This adds a virtio-net device to the VM, and redirects all the network traffic o
 ```
 This is useful in combination with usermode networking stacks such as [gvisor-tap-vsock](https://github.com/containers/gvisor-tap-vsock).
 
+See [this shell script](https://github.com/nirs/vmnet-helper/blob/main/examples/vfkit.sh) for an example of networking using `vmnet-helper`.
```

Comment:
forgot to add a link to contrib/scripts/start-gvproxy.sh in the next commit, I’ll do that tomorrow.